osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -383,7 +384,8 @@ protected override void Update()
                 new Dimension(GridSizeMode.Relative, 0.5f, minSize: 500, maxSize: 700 + widescreenBonusWidth * 300),
             };
 
-            updateDebounce();
+            if (this.IsCurrentScreen())
```

Comment:
What's this about? I don't like `TestFilteringRunsAfterReturningFromGameplay` failing just after this gets added. I'm not sure I recall it ever failing before. I'm not sure it's related, but it might be?

Reply:
I added this to fix tests, figured something something different order of execution (try removing, will get assertion failures on disabled bindable operations, 100% repro).

> I don't like TestFilteringRunsAfterReturningFromGameplay failing just after this gets added.

I think it's failing on other branches too 😦 tried to fix last night with a [commit to master](https://github.com/peppy/osu/commit/71d0afd4c2b3c7fe6e92558b5ecd849f67cb6384) but it's failing in a different place to expected..

Reply:
Well if the test is failing somewhere else already then I guess it's probably fine......

I guess next immediate order of business for me would be to go fix flaky tests on master because we're at the stage where it's unclear whether 4 reds is signal or noise.